server/client.go

```diff
@@ -1938,7 +1980,8 @@ func (c *client) canSubscribe(subject string) bool {
 
 	allowed := true
 
-	// Check allow list. If no allow list that means all are allowed. Deny can overrule.
+	// Check allow list. If no allow list that means all are
+	// allowed. Deny can overrule.
```

Comment:
Not sure why inserted a line break.

Reply:
Reverted this change, I had done fill-region over the comment